frontend/src/views/account-mgmt/ManageFacilityView.vue

```diff
@@ -342,9 +354,34 @@ export default {
       this.setWarningAlert('This feature is not yet implemented')
     },
 
+    /**
+     * Toggle expansion panels
+     */
     togglePanels() {
       this.panel = this.panel.length === 0 ? this.allLicenceIDs : []
     },
+
+    /**
+     * Handle edit mode change for expense authorities
+     */
+    editModeChangeExpAuth(editMode) {
+      this.disableEditAdditionalContacts = (editMode === true) ? true : false
+    },
+
+    /**
+     * Handle edit mode change for additional contacts
+     */
+    editModeChangeAddContacts(editMode) {
+      this.disableEditExpenseAuthorities = (editMode === true) ? true : false
+    },
```

Comment:
Code can be removed.

Reply:
Oh geez, shouldn't have missed this. Fixed.